README.md

```diff
@@ -1,6 +1,8 @@
 DIGImend kernel drivers
 =======================
 
+[![Build Status](https://travis-ci.org/DIGImend/digimend-kernel-drivers.svg?branch=master)](https://travis-ci.org/DIGImend/digimend-kernel-drivers)
```

Comment:
Instead of starting our README.md with a bunch of technical garbage like this, I think it's better to try to shove this into references at the bottom.

Reply:
Also, commit subject would be better/more correct English if written like `Add build status badges to README`. Next, it's just one badge, not "badges". Finally, following the "tagged" approach, it can become: `README.md: Add build status badge`